### ISSUE TITLE
Start a weekly/monthly development POV update. 

### ISSUE THREAD

**Issue by YalorMewn** [MetaFam/MetaGame#9]
Something for @MetaFam/combat-mages-devs to keep track of the progress of the platform, with insights from developers distilled down into information that is readable in plain English or images for non-technical contributors. 

See [here ](https://progress.swarm.city/) for an example of how this can be done well.  

**Could be any of the following**:
Graphic, a blog post, a story line, anything that brings people up to date in a fun way.

**Comment by yeqbfgxjiq** [comment]
The link might be broken

**Comment by YalorMewn** [comment]
Thanks @burrrata just fixed link !

**Comment by yeqbfgxjiq** [comment]
So, if I were to read between the lines on this.... are you suggesting that MetaGame could/should/would have a newsletter for weekly updates similar to the [Shots Fired](https://dapp.substack.com/) MetaCartel newsletter? 

> Note: this would not be as fancy as the Swarm thing, but it would be something we could launch **now** with minimal overhead and maintenance costs. Also, due to minimized complexity/overhead/[bus-factor](https://en.wikipedia.org/wiki/Bus_factor), anyone could easily contribute to it so it could be passed down like a sacred heirloom from generation to generation within the MetaGame fam.
